### PR TITLE
Start page

### PR DESCRIPTION
**Pull request by BoriskinaAnna**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/BoriskinaAnna/react-js.git"
+  },
+  "keywords": [],
+  "author": "Hanna Baryskina",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/BoriskinaAnna/react-js/issues"
```
Comment (AndreiShenets) [inline package.json]: Something wrong with url?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  "homepage": "https://github.com/BoriskinaAnna/react-js#readme"
```
Comment (AndreiShenets) [inline package.json]: Something wrong with url?

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>React App</title>
```
Comment (AndreiShenets) [inline public/index.html]: Need to change title in future

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (AndreiShenets) [inline public/index.html]: Language shouldn't be constant here

File: src/js/InformationAbouteLabs.json [hunk 5]
```diff
@@ -0,0 +1,97 @@
+{
```
Comment (AndreiShenets) [inline src/js/InformationAbouteLabs.json]: Why PascalCaseNaming?

File: src/js/InformationAbouteLabs.json [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+  "startedLabs": [
+    {
+      "id": 34,
+      "trainingDate": "07.2018 - 08.2018",
+      "trainingName": "Full-Stack js Students lab",
+      "admissionDate": "03.2018 - 06.2018",
+      "image": "/img/media/microsoft_net_logo.svg"
```
Comment (AndreiShenets) [inline src/js/InformationAbouteLabs.json]: Discuss why url here

File: src/js/components/App.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+import LabsList from 'js/components/labsList/labsList';
+import labs from "js/InformationAbouteLabs.json";
+import { translate } from 'react-i18next';
+import 'bootstrap/dist/css/bootstrap.css';
+import Header from "js/components/header/header";
+import Footer from "js/components/footer/footer";
+
+
+export class App extends Component {
+
```
Comment (AndreiShenets) [inline src/js/components/App.js]: Why spaces here?

File: src/js/components/App.js [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+
+
+    render() {
+
+        return(
+            <div>
+                <Header/>
+                <LabsList labs={labs.startedLabs} current={true}/>
```
Comment (AndreiShenets) [inline src/js/components/App.js]: Current is suspicious

File: src/js/components/footer/scss/styleFooter.css [hunk 9]
```diff
@@ -0,0 +1,20 @@
+footer {
```
Comment (AndreiShenets) [inline src/js/components/footer/scss/styleFooter.css]: Why not scss? Code style

File: src/js/components/footer/scss/styleFooter.scss [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+  .footer__phonesTitle {
+    color: rgb(246, 246, 245);
+  }
+  .footer__languages{
+    color: rgb(246, 246, 245);
+    cursor: pointer;
+  }
+}
+
```
Comment (AndreiShenets) [inline src/js/components/footer/scss/styleFooter.scss]: trailing new lines

File: src/js/components/header/header.js [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+class Header extends Component {
+
+
+    render(){
+        const {t} = this.props;
+        return (
+            <header>
+                <div className="container">
+                    <div className="row">
+                        <div className="col-6"><a className = "logo" href = ""/></div>
```
Comment (AndreiShenets) [inline src/js/components/header/header.js]: unnecessary spaces

File: src/js/components/i18n.js [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+    }
+};
+
+
+options.resources = {
+
+    en: {
+        translations: {
+            "btnAuthorization": "Log in",
+            "id": 1234,
```
Comment (AndreiShenets) [inline src/js/components/i18n.js]: id???

File: src/js/components/lab/lab.js [hunk 13]
```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react';
+import 'js/components/lab/scss/style.css';
+import { translate } from 'react-i18next';
+
+class Lab extends Component {
+
+
+    render(){
+        const {lab, current, t} = this.props;
+        this.state = {
```
Comment (AndreiShenets) [inline src/js/components/lab/lab.js]: state usually sets in constructor. also usually setState is used

File: src/js/components/lab/lab.js [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+            isCurrent: current
+        };
+        const labStatus = !this.state.isCurrent &&
+                <div className="labContent__labStatus">&nbsp;</div>
+
```
Comment (AndreiShenets) [inline src/js/components/lab/lab.js]: lines

File: src/js/components/lab/lab.js [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+class Lab extends Component {
+
+
+    render(){
+        const {lab, current, t} = this.props;
+        this.state = {
+            isCurrent: current
+        };
+        const labStatus = !this.state.isCurrent &&
+                <div className="labContent__labStatus">&nbsp;</div>
```
Comment (AndreiShenets) [inline src/js/components/lab/lab.js]: nbsp? what for?

**Review by AndreiShenets — CHANGES_REQUESTED** [review]
Ok in general but need to discuss some moments on Monday

**Comment by ghost** [comment]
There is no information about solution deployment in the readme file

**Pull request merged**